### PR TITLE
Manage 451 error status code - pn-4285

### PR DESCRIPTION
**Pull request by AndreaCimini90**
## Short description
Show a custom error message when exchangeToken returns a 451 error status code

## List of changes proposed in this pull request
- Added in user.utility.ts the case 451
- Added new class for the new error
- Added error localization
- Added 451 case in tests

## How to test
For pa, pf and pg try to login with a tax outside the white list and check that user cannot login and the correct message is shown; for test I excluded DSRDNI00A01A225I (dino - pf) and WZCCLV58R49B051K (m.messina - pa)

### CODE REVIEW

File: packages/pn-commons/src/utils/AppError/GenericAppError/UnavailableForLegalReasonsError.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  constructor(error: ServerResponseError) {
+    super(error);
+  }
+
+  getMessage() {
+    return {
+      title: getLocalizedOrDefaultLabel(
+        'common',
+        'errors.unavailable_legal_reasons.title',
+        'Paittaforma non operativa'
```
Comment (ZorbaDimatteo) [inline packages/pn-commons/src/utils/AppError/GenericAppError/UnavailableForLegalReasonsError.ts:15]: there's a typo: Piattaforma

File: packages/pn-commons/src/types/AppResponse.ts [hunk 2]
```diff
@@ -48,22 +48,23 @@ export enum ServerResponseErrorCode {
...
+  BAD_REQUEST_ERROR = 'BAD_REQUEST_ERROR',
   UNAUTHORIZED_ERROR = 'UNAUTHORIZED_ERROR',
   FORBIDDEN_ERROR = 'FORBIDDEN_ERROR',
+  UNAVAILABLE_FOR_LEGAL_REASONS_ERROR = 'UNAVAILABLE_FOR_LEGAL_REASONS_ERROR',
   NOT_FOUND_ERROR = 'NOT_FOUND_ERROR',
   INTERNAL_SERVER_ERROR = 'SERVER_ERROR',
 
   UNHANDLED_ERROR = 'UNHANDLED_ERROR',
-  GENRIC_ERROR = 'GENRIC_ERROR'
+  GENRIC_ERROR = 'GENRIC_ERROR',
```
Comment (ZorbaDimatteo) [inline packages/pn-commons/src/types/AppResponse.ts:62]: there's a typo: GENERIC

Comment (AndreaCimini90) [reply packages/pn-commons/src/types/AppResponse.ts]: done

File: packages/pn-commons/src/utils/AppError/GenericAppError/UnavailableForLegalReasonsError.ts (hunk 1 above)
Comment (AndreaCimini90) [reply packages/pn-commons/src/utils/AppError/GenericAppError/UnavailableForLegalReasonsError.ts]: done

**Review by ZorbaDimatteo — APPROVED** [review]

**Pull request merged**